### PR TITLE
Accelerator Rounds for sniper rifle

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
The rounds start very weak, but gains damage the farther it flies. 

Rewards long distance shots.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
I-I thought you were joking 
never forget dreamstation 2015-2015

**Comment by WJohn** [comment]
This is very cool sounding.

**Comment by Anonus** [comment]
Oh god one-hit-killing the captain down dream's hallways
I love it

**Comment by GunHog** [comment]
Remie Rifles are the new hot topic!

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 1]
```diff
@@ -153,4 +153,37 @@
...
-	breakthings = FALSE
\ No newline at end of file
+	breakthings = FALSE
+
+
+
+//Accelerator ammo
+
+/obj/item/ammo_box/magazine/sniper_rounds/accelerator
+	name = "sniper rounds (penetrator)"
```
Comment (tkdrg) [inline code/modules/projectiles/guns/projectile/sniper.dm]: `(penetrator)` ??????

**Comment by tkdrg** [comment]
VECTORS

**Pull request merged**